production/ksonnet/loki/config.libsonnet

```diff
@@ -42,6 +42,11 @@
 
     jaeger_reporter_max_queue: 1000,
 
+    // This is inserted into the gateway Nginx config file
+    // under the server directive
+
+    server_snippet: '',
```

Comment:
Couple of suggestions:
1. Please move `server_snippet` into `loki/gateway.libsonet` _config as it's used only on the nginx gateway now.
2. `server_snippet` -> `gateway_server_snippet` may be a better name?

Reply:
@kavirajk thanks for the hints, I updated gateway configuration accordingly